static/js/components/tiles/bivariate_tile.tsx

```diff
@@ -154,6 +145,32 @@ export function BivariateTile(props: BivariateTilePropType): JSX.Element {
   );
 }
 
+/**
+ * Returns callback for fetching chart CSV data
+ * @param props Chart properties
+ * @returns Async function for fetching chart CSV
+ */
+function getDataCsvCallback(
+  props: BivariateTilePropType
+): () => Promise<string> {
+  return () => {
+    // Assume all variables will have the same date
+    const date = getFirstStatVarSpecDate(props.statVarSpec);
```

Comment:
Kept data fetch as is since it already handles case where different variables have different dates. Added TODO here to handle multiple dates in the CSV export